pkg/commands/git_commands/stash.go

```diff
@@ -109,3 +109,8 @@ func (self *StashCommands) SaveStagedChanges(message string) error {
 
 	return nil
 }
+
+func (self *StashCommands) StashUntrackedChanges(message string) error {
```

Comment:
`StashChangedIncludingUntracked` is a little more correct

Reply:
Ok I've changed it now to `StashIncludeUntrackedChanges` to be consistent with `LcStashIncludeUntrackedChanges`.

Reply:
But I can change those names again if you'd prefer.